libafl/src/feedbacks/stacktrace.rs

```diff
@@ -0,0 +1,72 @@
+//! The StacktraceFeedbacks uses the backtrace hash and a hashset to only keep novel cases
+
+use hashbrown::HashSet;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    bolts::tuples::Named,
+    events::EventFirer,
+    executors::ExitKind,
+    feedbacks::Feedback,
+    inputs::Input,
+    observers::{ObserversTuple, StacktraceObserver},
+    state::HasClientPerfMonitor,
+    Error,
+};
+
+/// A [`StacktraceFeedback`] reduces the timeout value of a run.
```

Comment:
The comment seems outdated,
Maybe something similar to above (`The StacktraceFeedbacks uses the backtrace hash and a hashset to only keep novel cases`)

Reply:
Missed this one, will update